### PR TITLE
Update read me file

### PR DESCRIPTION
**Pull request by alefaissal**
# Update read me file
- Still need to add screenshots
- More features of the app (when ready and all merged)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+$ cd ios
+$ pod install
+$ cd ..
+```
+
+### Case you `don't have` neither npm or yarn check the website bellow:
+
+https://www.npmjs.com/get-npm
+https://yarnpkg.com/lang/en/docs/install/#mac-stable
+
```
Comment (siddharthparmar7) [inline README.md]: Just use yarn. No need to mention npm. Also, remove the link to install yarn.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+### Run (from client)
+
+```bash
+react-native run-ios
+```
+
+### To run this, `Xcode` from Apple Store must be previously installed
+
+---
```
Comment (siddharthparmar7) [inline README.md]: No need for this section.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
 # BC Parks Foundation
 
+BC Parks Foundation is a Mobile App build with React Native Technology that can be deployed in both ios and Android (with some ajustments) platforms.
+
+## <img src="./client/js/assets/images/discoverWhite.png" >
+
+## Some Screenshots from Mobile App:
```
Comment (siddharthparmar7) [inline README.md:11]: Please add only one gif showing some of your app's screens.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,62 @@
 # BC Parks Foundation
 
+BC Parks Foundation is a Mobile App build with React Native Technology that can be deployed in both ios and Android (with some ajustments) platforms.
+
+## <img src="./client/js/assets/images/discoverWhite.png" >
```
Comment (siddharthparmar7) [inline README.md:9]: This image is not loading properly. Please check.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,62 @@
...
+
+## Some Screenshots from Mobile App:
+
+## Description
+
+BC Parks Foundation is a Mobile App to connect people to Outdoor hosted events or selfguided tours, where is possible to buy your ticket, save your favorite events/tours or get in touch with one the hosts.
```
Comment (siddharthparmar7) [inline README.md]: The description should be the first section of your README. Please move it.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the requested changes.

File: README.md (hunk 5 above)
Comment (alefaissal) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (alefaissal) [reply README.md:9]: Is loading for me

File: README.md (hunk 3 above)
Comment (alefaissal) [reply README.md:11]: We'll do when we finalize the app

File: README.md (hunk 2 above)
Comment (alefaissal) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (alefaissal) [reply README.md]: Removed

**Pull request closed without merging**